plugins/sip/services/PexipService.php

```diff
@@ -102,21 +111,26 @@ public function handleIncomingCallAction()
 		}
 		catch(Exception $e)
 		{
-			KalturaLog::err('Error validating and retrieving Entry for sip Call');
+			$msg = 'Error validating and retrieving Entry for sip Call';
```

Comment:
Same here - all errors in one SIP ERROR file

Reply:
all msgs were already created. 
i just add the mag sending to the response object. there is no need to refactor all msg codes for that purpose since this is not something that is going to change or retranslated.